.github/workflows/python-lint.yaml

```diff
@@ -15,14 +15,9 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.10.13"
-      - run: pip install -r ./requirements.txt
-        working-directory: ./backend
-      - run: black --check --diff ./
-        working-directory: ./backend
-      - run: isort --profile black --check --diff ./
-        working-directory: ./backend
-      - run: mypy ./
-        working-directory: ./backend
-      - run: pylint --rcfile ./backend/pylintrc -E ./backend
-      - run: pytest ./
-        working-directory: ./backend
+      - run: pip install -r ./backend/requirements.txt
+      - run: black --check --diff ./backend/
+      - run: isort --profile black --check --diff ./backend/
```

Comment:
This was intentional. Isort behaves differently depending on if it's aware of pip dependencies and not, so currently isort is not correctly checking 3rd party library import order on the CI.

Reply:
Alright, I'll just leave this as-is and run isort in the broken mode. I'll figure out the issue later in-person.